example/package.json

```
@@ -1,24 +1,28 @@
 {
-  "version": "1.0.0",
-  "description": "Example App for the AppLovin MAX React Native Module for Android and iOS",
-  "homepage": "https://applovin.com",
+  "name": "react-native-applovin-max-example",
+  "version": "0.0.1",
```

I'd keep version the same, doesn't really matter, but just consistency
```suggestion
  "version": "1.0.0",
```